jobs/semantic-release/action.yaml

```diff
@@ -0,0 +1,83 @@
+name: semantic-release
+description: Release a new version of the project using conventional commits
+author: Lenra-io
+branding:
+  color: orange
+  icon: tag
+
+inputs:
+  node-version:
+    description: The version of node to use
+    required: false
+    default: '18'
+  dry-run:
+    description: Whether to publish the release or not
+    required: false
+    default: 'false'
+  release-branches:
+    description: The branches on which releases should happen, comma separated name of branches
+    required: false
+    default: 'main'
+  prerelease-branches:
+    description: The branches on which prereleases should happen, comma separated name of branches
+    required: false
+    default: 'rc,beta,alpha'
+  assets:
+    description: 'The assets to upload to the release, comma separated list of `{"path": "<glob>", "name": "<name>"}` objects'
+    required: false
+    default: ''
+
+outputs:
+  release-published:
+    description: "Whether a release was published or not"
+    value: ${{ steps.semantic-release.outputs.new-release-published }}
+  release-version:
+    description: "The next version discovered by conventional commits"
+    value: ${{ steps.semantic-release.outputs.new-release-version }}
```

Comment:
Default value will be empty string, because no release will be published, there is no version.

I think this is the normal use. In case there is no new version, we can then just set a `0.0.0-dev.0` for example on local use. But CI didn't have to use it and will never build with semantic-release for PR workflow.

Reply:
I thought that the CI would fail the branch does not match, since you try to get the output of a job that won't run, but if it's not the case, this can be resolved